README.rst

```diff
@@ -68,52 +67,55 @@ Search Enhancements
 Detailed Documentation
 ======================
 
-A full Documentation of the Solr integration of Plone could be found on `collectivesolr.readthedocs.org`_.
+The full documentation of Plone integration with Solr can be found at `read the docs`_.
 
-.. _`collectivesolr.readthedocs.org`: http://collectivesolr.readthedocs.org/en/latest/
+.. _`read the docs`: https://collectivesolr.readthedocs.io/en/latest/
```

Comment:
@gforcada I kind of prefer to keep the full URL. Some people might not know what readthedocs is.